libmscore/tremolo.cpp

```diff
@@ -513,6 +481,48 @@ void Tremolo::layoutTwoNotesTremolo(qreal x, qreal y, qreal h, qreal _spatium)
       setPos(x, y + beamYOffset);
       }
 
+      
+//---------------------------------------------------------
+//   layout
+//---------------------------------------------------------
+
+void Tremolo::layout()
```

Comment:
Any reason you moved this whole function? This makes the diff larger, obscures what has actually changed, and reduces the benefit of `git blame`, all for no purpose.

Reply:
This is because I want to make the two functions for single-note and double-note tremolos respectively be adjacent, rather than having this main `layout()` function in the way.